temporalio/lib/temporalio/priority.rb

```diff
@@ -4,7 +4,9 @@
 
 module Temporalio
   Priority = Data.define(
-    :priority_key
+    :priority_key,
+    :fairness_key,
+    :weight
```

Comment:
You called this `fairness_weight` in Python

Reply:
Yeah, dumb oversight